src/Controls/src/Core/Brush.cs

```diff
@@ -10,7 +10,7 @@ public abstract partial class Brush : Element
 		/// <include file="../../docs/Microsoft.Maui.Controls/Brush.xml" path="//Member[@MemberName='Default']/Docs/*" />
 		public static Brush Default
 		{
-			get { return new SolidColorBrush(null); }
+			get { return new ImmutableBrush(null); }
```

Comment:
Apparently, you could store your favorite color in `Brush.Default.Color = Colors.CornflowerBlue`.